docs/reference/analysis/analyzers/lang-analyzer.asciidoc

```diff
@@ -421,6 +421,9 @@ PUT /catalan_example
 [[cjk-analyzer]]
 ===== `cjk` analyzer
 
+NOTE: You may find that `icu_analyzer` in the ICU analysis plugin works better
+for CJK text than the `cjk` analyzer.
```

Comment:
Maybe add something like "experiment with your text and queries" or something like that.